docs/variants/dell_optiplex/initial-deployment.md

```diff
@@ -125,17 +103,20 @@ switching to open-source firmware.
   Reading flash... done.
   ```
 
-If you will face any issues please refer to [troubleshooting section](#troubleshooting).
+If you face any issues, please refer to the
+[troubleshooting section](#troubleshooting).
 
 ## Get Dasharo
 
-Download the Dell OptiPlex 7010/9010 Dasharo from [release section](releases.md#binaries)
-or [build from source](building-manual.md).
+Download the Dell OptiPlex 7010/9010 Dasharo from the
+[release section](releases.md#binaries) or
+[build from source](building-manual.md).
+
 
 ## Non-redistributable blobs
 
-Some Dell OptiPlex 7010/9010 SFF binary blobs have unknown license or
-questionable redistributability. To avoid potential issues we transfer
+Some Dell OptiPlex 7010/9010 SFF binary blobs have an unknown license or
+questionable redistribution policy. To avoid potential issues, we transfer
 non-redistributable components from your original BIOS to Dell OptiPlex 9010
 Dasharo binary using [fwdeploy](https://github.com/Dasharo/fwdeploy).
```

Comment:
Is it up-to-date mentioning `fwdeploy` here? I remember I could not really use it.

Reply:
@macpijan can we use DTS for that? I can try to retest initial deployment if needed, but TBH ideally would be if I could do that with DTS. Also please note we improved fwdeploy in form of Dasharo Blobs Transmission script. I'm pretty sure I didn't tested initial deployment for long time. We just have to agree what synergy we can get now from DTS and what have to be done externally and does it make sense to fix it now or we will get fixes from DTS sooner, so the documentation could be fixed.

Also as part of OptiPlex Tuesday we can take care about DTS part of it.

Reply:
so we still ned to update this section

Reply:
@macpijan https://github.com/Dasharo/docs/pull/139/commits/f312a418169a77003a833dab9ca2cf3c2db4e606

Reply:
LGTM, DTS deploy script has been already tested and proven successful. Enough for me to close this discussion